CMakeLists.txt

```diff
@@ -142,6 +138,13 @@ set(THIRDPARTY_DIR ${PROJECT_SOURCE_DIR}/thirdparty)
 
 include(SetupBuildEnvironment)
 include(GetPlatformInfo)
+if(OS_IS_FBSD)
+    option(MUE_BUILD_CRASHPAD_CLIENT "Build crashpad client" OFF)
+else()
+    option(MUE_BUILD_CRASHPAD_CLIENT "Build crashpad client" ON)
+    set(MUE_CRASH_REPORT_URL "" CACHE STRING "URL where to send crash reports")
+    option(MUE_CRASHPAD_HANDLER_PATH "Path to custom crashpad_handler executable (optional)" "")
+endif()
```

Comment:
Please do it here https://github.com/musescore/MuseScore/blob/master/SetupConfigure.cmake#L104 
like:
```
if (OS_IS_FBSD)
    message(WARNING "...")
    set(MUE_BUILD_CRASHPAD_CLIENT OFF)
endif()
```